### DISCUSSION TITLE
Una pregunta

### DISCUSSION THREAD

**Discussion by mafer23** [Launch-X-Latam/MisionFrontEnd#460] [Team Posts]
Alguien sabe si las sesiones quedan en youtube  o se pueden descargar??? el internet esta muy malo en donde vivo

**Comment by NelsonOrtizLeal** [comment]
Las sesiones se quedan guardadas en la plataforma de teams, las puedes ver
dando click al enlace de la sesión o al final de la semana suben un link
con las grabaciones.(Grabaciones de stream de teams).

**Comment by mafer23** [comment]
el internet  no esta sirviendo para verla

**Comment by NelsonOrtizLeal** [comment]
Descuida las clases se quedan guardadas. Y tendrás acceso a los recursos
por 1 año o algo así. Hay una forma de descargar los videos usando Python
aunque no se que tan legal o si rompe alguna norma. Podrías hacer algunas
búsquedas en you tube con lo que te acabo de decir y encontraras un video
explicando. Aunque repito no se que tan seguro o si infringe alguna norma.
Lo que hagas es bajo tu propio riesgo.

**Comment by Jamesteven** [comment]
A mí tampoco me deja ver los clips desde las grabaciones

James Steven Rodríguez Acero


________________________________
From: Maria Fernanda Palencia ***@***.***>
Sent: Wednesday, October 26, 2022 11:00:57 AM
Cc: Subscribed ***@***.***>
Subject: [Launch-X-Latam/mision-frontend] Una pregunta (#30)


Alguien sabe si las sesiones quedan en youtube o se pueden descargar??? el internet esta muy malo en donde vivo

**Comment by AmeyalliRomeroM** [comment]
Hola Chic@s

Si se pueden ver los vídeos en la parte de grabaciones 
sino en cada canal, el Mission Comander pone una lista de todos los vídeos referentes.

Prueben desde la aplicación de Microsoft teams o abriendo desde el navegador en pestaña de incognito.

![image](https://user-images.githubusercontent.com/106411263/198176826-5f0cc73f-7299-4107-b3d8-bc7e740e7196.png)

![image](https://user-images.githubusercontent.com/106411263/198177460-ef1a1dc3-3dcf-409f-a6e6-b5f8df195837.png)

**Comment by AlexOrrala** [comment]
Hola buenas noches, tengo una duda acerca de como me puedo unir al teams? Muchas gracias.

El 26 oct. 2022 21:38, AmeyalliRomeroM ***@***.***> escribió:

Hola ***@***.***

Si se pueden ver los vídeos en la parte de grabaciones
sino en cada canal, el Mission Comander pone una lista de todos los vídeos referentes.

Prueben desde la aplicación de Microsoft teams o abriendo desde el navegador en pestaña de incognito.

[image]<https://user-images.githubusercontent.com/106411263/198176826-5f0cc73f-7299-4107-b3d8-bc7e740e7196.png>

[image]<https://user-images.githubusercontent.com/106411263/198177460-ef1a1dc3-3dcf-409f-a6e6-b5f8df195837.png>

**Comment by S3gn1x** [comment]
Hola a Todos yo no encuentro la parte de equipos del Launch X, como hago?, gracias
